python/dolfinx/fem/bcs.py

```diff
@@ -8,23 +8,22 @@
 
 from __future__ import annotations
 
+import collections.abc
 import typing
 
-if typing.TYPE_CHECKING:
```

Comment:
We need to check that these changes work with earlier version of Python. This particular change might be ok - it was needed for doc generation with earlier version of Python (v3.8 and earlier (?)).

Reply:
I'll run a wheel build which includes Python 3.8 https://github.com/FEniCS/dolfinx/runs/6589671237?check_suite_focus=true

Reply:
I can confirm that all tests pass on Python 3.8 with this branch.